resources/default_contacts.json

```diff
@@ -204,5 +204,18 @@
         {
             "en": "http://test.etherplay.io"
         }
+    },
+      "REXmls":
+    {
+        "name":
+        {
+            "en": "REXmls"
+        },
+        "photo-path": "contacts://rexmls",
+        "dapp?": true,
+        "dapp-url":
+        {
+            "en": "http://http://rexmls.com/"
```

Comment:
Hey sorry about that extra http:// that was me being careless. We have not quite launched our dApp yet so I guess I will wait untill we go live to put the link to our dApp in your contacts. Thanks guys!!